test/test-data-unicode-properties.json

```diff
@@ -182,13 +182,13 @@
   "\\p{}": {
     "type": "error",
     "name": "SyntaxError",
-    "message": "Expected atom at position 2\n    \\p{}\n      ^",
+    "message": "atomEscape at position 1\n    \\p{}\n     ^",
     "input": "\\p{}"
   },
   "\\P{}": {
     "type": "error",
     "name": "SyntaxError",
-    "message": "Expected atom at position 2\n    \\P{}\n      ^",
+    "message": "atomEscape at position 1\n    \\P{}\n     ^",
```

Comment:
Just spotted (not caused by this PR): Is there any good reason why "Expected atom at [...]" is used and in the other case it's "atomEscape at [...]"?

Would that make sense to add an "Expected" to the bails in [1]?

+\cc @mathiasbynens 

[1]: https://github.com/jviereck/regjsparser/blob/2cf92b1ee00ea2a8c90198e7dcbb379fa7dbc2c1/parser.js#L1043